shopping-list-web/src/InputText/InputTest.test.tsx

```diff
@@ -1,6 +1,7 @@
 import { fireEvent, render, screen, waitFor } from "@testing-library/react";
 import InputText from "./InputText";
 
+const mockCurrentProduct = ""
```

Comment:
🤔 can you not mock it with jest like the others are? Impressed you touched the tests though tbh